app/views/news/_news.html.erb

```diff
@@ -1,17 +1,36 @@
-<div class="news">
-  <h4>
-    <%= link_to(news, news_path(news)) %>
-    <% if news.image.present? %>
-      <%= fa_icon('image') %>
+<% pinned |= false %>
+<div class="news start box">
+  <div class="headline">
+    <h3><%= news.title %></h3>
+
+    <% if pinned %>
+      <div class="pinned pull-right">
+        <%= fa_icon('thumb-tack lg') %>
+      </div>
     <% end %>
-  </h4>
-  <%= truncate(markdown_plain(news.content),
-               length: 140,
-               separator: ''){ link_to(t('.read_more'), news_path(news))} %>
+  </div>
+
+  <div class="info">
+    <%= news_user(news) %>
+    <span class="date">
+      <%= l(news.created_at) %>
+    </span>
+  </div>
 
-   <%= news_url(news) %>
+  <% if news.image? %>
+    <div class="margin-bottom-20">
```

Comment:
The `markdown(news.content)` is repeated, put only image tag within if-statement instead :)